global_vars.tf

```diff
@@ -19,3 +19,31 @@ variable "submitted_responses_write_capacity" {
   description = "The number of Write units for the Submitted Responses table"
   default     = 25
 }
+
+variable "questionnaire_state_read_capacity" {
+  description = "The number of Read units for the Questionnaire State table"
+  default     = 25
+}
+
+variable "questionnaire_state_write_capacity" {
+  description = "The number of Write units for the Questionnaire State table"
+  default     = 25
+}
+variable "eq_session_read_capacity" {
+  description = "The number of Read units for the EQ Session table"
+  default     = 25
+}
+
+variable "eq_session_write_capacity" {
+  description = "The number of Write units for the EQ Session table"
+  default     = 25
+}
+variable "used_jti_claim_read_capacity" {
+  description = "The number of Read units for the Used JTI claims table"
+  default     = 25
```

Comment:
This can be low if not nothing as we don't read.
Will it allow `0`?

Reply:
Won't allow 0, have set it to 1